PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-vm/src/main/java/com/microsoft/azure/toolkit/intellij/vm/ssh/ConnectBySshAction.java

```diff
@@ -1,13 +1,70 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
 package com.microsoft.azure.toolkit.intellij.vm.ssh;
 
 import com.intellij.openapi.project.Project;
+import com.intellij.ssh.config.unified.SshConfig;
+import com.intellij.ssh.config.unified.SshConfigManager;
+import com.microsoft.azure.toolkit.lib.common.bundle.AzureString;
+import com.microsoft.azure.toolkit.lib.common.messager.AzureMessager;
+import com.microsoft.azure.toolkit.lib.common.task.AzureTask;
+import com.microsoft.azure.toolkit.lib.common.task.AzureTaskManager;
 import com.microsoft.azure.toolkit.lib.compute.virtualmachine.VirtualMachine;
+import org.jetbrains.plugins.terminal.ShellTerminalWidget;
+import org.jetbrains.plugins.terminal.TerminalView;
 
 import javax.annotation.Nonnull;
 
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static com.microsoft.azure.toolkit.lib.common.operation.OperationBundle.description;
+
+/**
+ * connect to selected machine by SSH in Intellij Ultimate.
+ */
 public class ConnectBySshAction {
+    private static final String SSH_TERMINAL_TABLE_NAME = "SSH - %s";
+    private static final String CMD_SSH = "ssh %s@%s";
+    private static final String CMD_SSH_KEY_PAIR = "ssh %s@%s -i %s";
```

Comment:
it's OK to open the terminal manually in Community Edition.
we'd better reuse IntelliJ's own action (which is based on SSH configuration) in Ultimate Edition.